socs/agents/wiregrid_tiltsensor/drivers/dwl.py

```diff
@@ -0,0 +1,146 @@
+# Built-in python modules
+import time as tm
+
+# Control modules
+from socs.common import moxa_serial as mx
+
+
+class DWL:
+    """
+    The DWL object is for communicating with the DWL-5000XY gravity sensor
+
+    Args:
+    tcp_ip (str): TCP IP address
+    tcp_port (int): TCP port
+    """
+    waittime = 0.05  # sec
+
+    def __init__(self, tcp_ip=None, tcp_port=None, timeout=None, isSingle=False, verbose=0):
+        self.tcp_ip = tcp_ip
+        self.tcp_port = tcp_port
+        self.isSingle = isSingle
+        self.verbose = verbose
+
+        # Connect to device
+        msg = self.__conn(tcp_ip, tcp_port, timeout)
+        print(msg)
+
+    def __del__(self):
+        print(f"Disconnecting from TCP IP {self.tcp_ip} at port {self.tcp_port}")
+        self.ser.close()
+        return
+
+    def get_angle(self):
+        """ Measure the single-axis or two-axis angle """
+        self.clean_serial()
+        if self.isSingle:
+            command = b"\x06\x01\x01\xAA\x00\x00\x00\x00\x00\x00\x00\x00"
+        else:
+            command = b"\x06\x01\x02\xAA\x00\x00\x00\x00\x00\x00\x00\x00"
+        if self.verbose > 0:
+            print(f'get_angle() command = {command}')
+
+        read = []
+        SIZE = 12
+
+        # write and read serial
+        self.ser.write(command)
+        read_hex = self.ser.read(SIZE)
+        if self.verbose > 0:
+            print(f'{read_hex}')
```

Comment:
This gets printed again anyway on line 53. Any reason to keep this? If not, remove it.

Reply:
I removed it. (022b0cc)